rocketpy/Flight.py

```diff
@@ -1340,11 +1342,25 @@ def uDot(self, t, u, postProcessing=False):
                     M1 -= (compCp + a) * compLiftYB
                     M2 += (compCp + a) * compLiftXB
             # Calculates Roll Moment
-            if aerodynamicSurface[-1] == "Fins":
-                Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface[2]
-                Clf = Clfdelta * cantAngleRad
-                Cld = Cldomega * omega3 / freestreamSpeed if freestreamSpeed != 0 else 0
-                M3 += Clf - Cld
+            if aerodynamicSurface["name"] == "Fins":
+                Clfdelta, Cldomega, cantAngleRad = aerodynamicSurface["roll parameters"]
+                M3f = (
+                    (1 / 2 * rho * freestreamSpeed ** 2)
+                    * self.rocket.area
+                    * 2
+                    * self.rocket.radius
+                    * Clfdelta(freestreamMach)
+                    * cantAngleRad
+                )
+                M3d = (
+                    (1 / 2 * rho * freestreamSpeed)
+                    * self.rocket.area
+                    * 2
```

Comment:
Based on equation (25) of your "Roll_Equations.pdf", shouldn't be a 1/2 (i.e. 0.5) instead of  2?

Reply:
Actually, I think it's missing parentesis. Should be `* (2* self.rocket.radius)**2`

Reply:
> Should be `* (2* self.rocket.radius)**2`

Maybe?

![image](https://user-images.githubusercontent.com/69485049/153711166-56bb7df6-de48-40eb-92d6-a7b0dbfe48fb.png)

Doing it from scratch:
```
M3d = (1/2 * rho * freestreamSpeed**2) 
             * self.rocket.area 
             * (2 * self.rocket.radius) 
             * Cldomega(freestreamMach) 
             * omega3
             * (2 * self.rocket.radius)
             / (2 * freestreamSpeed)
```

Simplifying:
```
M3d = (1/2 * rho * freestreamSpeed) 
             * self.rocket.area 
             * 2
             * self.rocket.radius**2 
             * Cldomega(freestreamMach) 
             * omega3
```

I think if we were to do it as you suggested it we would have to divide by 2:

```
M3d = (1/2 * rho * freestreamSpeed) 
             * self.rocket.area 
             * (2 * self.rocket.radius)**2 
             * Cldomega(freestreamMach) 
             * omega3
             / 2
```
I think this is better for readability though, I'll leave it like this